lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -164,7 +164,11 @@ def add_copy_resources_script_phase
             phase = create_or_update_build_phase(native_target, phase_name)
             script_path = target.copy_resources_script_relative_path
             phase.shell_script = %("#{script_path}"\n)
-            # TODO: Add input and output paths for the copy resources script phase.
+            resource_paths_by_config = target.resource_paths_by_config
+            unless resource_paths_by_config.empty?
+              phase.input_paths = [target.copy_resources_script_relative_path, *resource_paths_by_config.values.flatten.uniq]
+              phase.output_paths = ['${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}']
```

Comment:
@dnkoutso This looks fantastic, I'm trying it out. Unfortunately, it looks like this script is being executed on a no-op build (build twice in a row). What little I can find on this feature is here https://lists.apple.com/archives/xcode-users/2009/Oct/msg00316.html

>Adding the directory as an input works for me, but only if the files in that directory are 1) in the Xcode project, and 2) modified from within Xcode, rather than from the terminal or another application.